.github/CODEOWNERS

```diff
@@ -1,17 +1,20 @@
 *                                                  @plexoos           @nigmatkulov
 /OnlTools                                          @kehw              @plexoos          @jml985
 /StDb                                              @dmarkh            @plexoos
-/StDb/idl                                          @akioogawa @dmarkh         @fgeurts          @iraklic @plexoos        @yfisyak
+/StDb/idl                                          @akioogawa 	      @dmarkh		@fgeurts	@iraklic	@plexoos	@yfisyak
+/StRoot/GenFit                                     @yfisyak
+/StRoot/KFParticle*                                @yfisyak
```

Comment:
Our repository does not contain these subdirectories

Reply:
Thanks, Dmitri. I removed these two lines.